bee-network/bee-autopeering/src/discovery/query.rs

```diff
@@ -13,7 +13,7 @@ use crate::{
     task::Repeat,
 };
 
-use rand::{thread_rng, Rng};
+use rand::{thread_rng, Rng as _};
```

Comment:
Why do we need the rename here?

Reply:
haha, we had this discussion already a while back. My explanations for doing this are: 
(a) Because I know for sure, that I won't ever implement that trait in that module I am releasing my claim on that symbol, and whoever might change that module in the future and finding this symbol useful as a name is free to use it.
(b) consistency with the rest of this crate (because I did quite a few times).
(c) I find it useful in terms of readability, because it's a statement to the reader.
(d) I can, because clippy doesn't complain about it :trollface:

Reply:
If clippy doesn't complain I won't complain either 😁